plugins/python/python_plugin.c

```diff
@@ -1297,29 +1288,49 @@ void uwsgi_python_init_apps() {
 
 }
 
-void uwsgi_python_master_fixup(int step) {
 
-	static int master_fixed = 0;
-	static int worker_fixed = 0;
+void uwsgi_python_pre_uwsgi_fork() {
+	if (!uwsgi.master_process) return;
```

Comment:
is it possible for this to get called in a child process? or is this just a general safe-guard?